docs-source/content/userguide/managing-domains/domain-lifecycle/liveness-readiness-probe-customization.md

```diff
@@ -0,0 +1,80 @@
+
+**Note**: The liveness probe custom script option is for advanced usage only and its value is not set by default. If the specified script is not found, then custom script is ignored and existing liveness script will perform its normal checks.
+
+**Note**: Oracle recommends against having any long running calls (e.g. any network calls or executing wlst.sh) in the liveness probe custom script.
+
+Use the following configuration to specify liveness probe custom script using `livenessProbeCustomScript` domain resource field.
+```
+spec:
+  livenessProbeCustomScript: /u01/customLivenessProbe.sh
+```
+
+Use the following configuration to specify liveness probe custom script using `LIVENESS_PROBE_CUSTOM_SCRIPT` environment variable.
+```
+    serverPod:
+      env:
+      - name: LIVENESS_PROBE_CUSTOM_SCRIPT
+        value: /u01/customLivenessProbe.sh
+```
+
+Following environment variables are available for use in the liveness probe custom script which will be invoked by `livenessProbe.sh`.
+
+`ORACLE_HOME` or `MW_HOME`: The Oracle Fusion Middleware software location as a file system path within the container.
+
+`WL_HOME`: The Weblogic Server installation location as a file system path within the container. 
+
+`DOMAIN_HOME`: The domain home location as a file system path within the container.
+
+`JAVA_HOME`: The Java software installation location as a file system path within the container.
+
+`DOMAIN_NAME`: The WebLogic Server domain name.
+
+`DOMAIN_UID`: The domain unique identifier.
+
+`SERVER_NAME`: The WebLogic Server instance name.
+
+`LOG_HOME`: The WebLogic log location as a file system path within the container. This variable is available only if its value is set in the configuration.
+
+#### Readiness probe customization
+
+The readiness probe is configured to use the WebLogic Server ReadyApp framework. The readiness probe is used to determine if the server is ready to accept user requests. The readiness is used to determine when a server should be included in a load balancer's endpoints, when a restarted server is fully started in the case of a rolling restart, and for various other purposes. The readiness probe is by default configured to check readiness every 5 seconds, and to timeout after 5 seconds. 
+
+The readiness probe interval and timeout can be customized using the `readinessProbe` attribute under the `serverPod` element of the domain resource.
+
+Following is an example configuration to change liveness probe interval and timeout value.
+```
+  serverPod:
+    readinessProbe:
+      periodSeconds: 10
+      timeoutSeconds: 10
```

Comment:
The readiness probe relies on calling a WebLogic Server's built-in readiness application, and this application can be customized if needed. This is rarely needed, but I think we have one user that's doing this - Johnny has the details.